modules/apps/foundation/address-formatter/address-formatter-api/src/main/java/com/liferay/address/formatter/BaseAddressFormatter.java

```diff
@@ -37,7 +43,7 @@ public String getCountry(Address address) {
 		Country country = countryService.fetchCountry(countryId);
 
 		if (country != null) {
-			return country.getName();
+			return country.getName(getUserLocale());
```

Comment:
Hey Jorge,

I think you might have misread the comparison operator. It's checking for `(country != null)`, so if the address has a country field, we display the country name in user's locale. We need to pass a locale object here because `country` is a localized field. If we don't specify a locale, the country will be printed as keys (for example, united-states instead of United States).

Also to note that we are not translating the address to user's locale because the address object is not localized, only the country field is localized.